index.html

```diff
@@ -1,138 +1,215 @@
-                <ul class="list">
-                    <li class="item">css</li>
-                    <li class="item">html</li>
-                    <li class="item">bootstrap</li>
-                    <li class="Ruby">Ruby</li>
-                </ul>
-                <button type="button" class="btn">See Project</button>
-            </div>
+        <div class="holder">
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="list">
+            <li class="item">css</li>
+            <li class="item">html</li>
+            <li class="item">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
         </div>
-        <div class="background4 background">
-            <div class="stories1">
-                <img src="images/placeholder.png" alt="placeholder">
-            </div>
-            <div class="holder">
-                <h2>Multi-Post Stories</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <ul class="list">
-                    <li class="item">css</li>
-                    <li class="item">html</li>
-                    <li class="item">bootstrap</li>
-                    <li class="Ruby">Ruby</li>
-                </ul>
-                <button type="button" class="btn">See Project</button>
-            </div>
+      </div>
+      <div class="background4 background">
+        <div class="stories1">
+          <img src="images/placeholder.png" alt="placeholder" />
         </div>
+        <div class="holder">
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="list">
+            <li class="item">css</li>
+            <li class="item">html</li>
+            <li class="item">bootstrap</li>
+            <li class="Ruby">Ruby</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
+        </div>
+      </div>
     </section>
+    <section>
+        <div class="contact">
 
-</body>
-</html>
\ No newline at end of file
+            <h1>CONTACT US</h1>
+            <p>
+              If you have an application you are interested in developing, a feature
+              that you need built or a project that needs coding. I'd love to help
+              with it
+            </p>
```

Comment:
Great job team 🤝

- [ ] [Optional] You may try to make the font family of your text throughout the whole project a replica of the one provided in the Figma design.